tests/test_create.py

```diff
@@ -2026,6 +2026,32 @@ def test_conda_pip_interop_compatible_release_operator(self):
                 run_command(Commands.INSTALL, prefix, "-c", "https://repo.anaconda.com/pkgs/free",
                             "agate=1.6", "--dry-run")
 
+    def test_conda_recovery_of_pip_inconsistent_env(self):
+        with make_temp_env("pip=10", "python", "anaconda-client",
+                           use_restricted_unicode=on_win) as prefix:
+            run_command(Commands.CONFIG, prefix, "--set", "pip_interop_enabled", "true")
```

Comment:
will this test only work if pip_interop is enabled?

Reply:
I think so.  If it doesn't then the precs for the pip installed stuff won't be in the collection of prefix_records, if I understand correctly.